build/common.sh

```diff
@@ -851,3 +852,20 @@ function kube::release::gcs::copy_release_artifacts() {
 
   gsutil ls -lhr "${gcs_destination}"
 }
+
+function kube::release::gcs::publish_latest() {
+  local latest_file_dst="gs://${KUBE_GCS_RELEASE_BUCKET}/${KUBE_GCS_LATEST_FILE}"
```

Comment:
We probably want to put this under `gs://${KUBE_GCS_RELEASE_BUCKET}/${KUBE_GCS_RELEASE_PREFIX}.  That is the "series" of release and the real root of the release tree.